build.gradle

```diff
@@ -3,6 +3,7 @@ buildscript {
 
     repositories {
         jcenter()
+        maven { url 'https://maven.google.com' }
```

Comment:
Why don't use  `google()` ?

Reply:
Didn't know that there is an equivalent for that, thanks :)